src/pudl/metadata/classes.py

```diff
@@ -1971,19 +1977,18 @@ def from_data_source_ids(
             output_path: PUDL_OUTPUT path.
             data_source_ids: ids of data sources currently included in Datasette
             xbrl_ids: ids of data converted XBRL data to be included in Datasette
-            extra_etl_groups: ETL groups with resources that should be included
+            exclude_intermediate_resources: exlude intermediate resources if True
         """
         # Compile a list of DataSource objects for use in the template
         data_sources = [DataSource.from_id(ds_id) for ds_id in data_source_ids]
 
         # Instantiate all possible resources in a Package:
-        pkg = Package.from_resource_ids()
-        # Grab a list of just the resources we want to output:
-        resources = [
-            res
-            for res in pkg.resources
-            if res.etl_group in data_source_ids + extra_etl_groups
-        ]
+        resources = Package.from_resource_ids().get_sorted_resources()
```

Comment:
The resources are correctly sorted in the metadata.yml file but it doesn't have an effect on the final ordering in the datasette UI. Any idea how to do this @zaneselvans?

Reply:
Looks like I might need to adjust the [HTML templates datasette uses.](https://docs.datasette.io/en/stable/custom_templates.html#custom-templates) I think it's currently just pulling the table names from the database directly: https://github.com/simonw/datasette/blob/2ff4d4a60a348c143f79d63c48c329ffd0c1f02f/datasette/templates/database.html#L50-L82

Reply:
The layout of Datasette pages (and the set of pages which exist) can be customized extensively with custom JavaScript, CSS, and [Jinja templates](https://docs.datasette.io/en/stable/custom_templates.html#custom-templates)

It seems like maybe we want a child-template that extends the database page that lists all of the tables, and sorts / filters them.

Reply:
And if we want to hide all of the `_out` tables, then it seems like we can add them to [the list of `hidden_tables`](https://docs.datasette.io/en/stable/metadata.html#hiding-tables) in the metadata that we generate.

Reply:
Seems like we need to create a directory of our own `templates` and create a `database-pudl.html` Jinja template that overrides the default `database.html` template, at least for the content block?